internal/docs/generated/fndocs/docs.go

```diff
@@ -96,6 +96,18 @@ var RunExamples = `
   kpt fn run DIR/
 `
 
+var DocShort = `Locally execuate a function to get the help text`
```

Comment:
Explain the WHAT not the HOW.

Something like:

`Displays the documentation for a function`